ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -158,6 +158,24 @@ Item {
               color: Theme.palette.directColor1
           }
 
+          MouseArea {
+              property bool accountPopupOpened: false
+
+              anchors.fill: parent
+              anchors.margins: -10
+              cursorShape: Qt.PointingHandCursor
+              hoverEnabled: true
+              onEntered: {
+                  accountPopupOpened = accountsPopup.opened
+              }
+              onPressed: {
+                  if (!accountPopupOpened) {
+                      changeAccountBtn.clicked(mouse)
+                  }
+                  accountPopupOpened = accountsPopup.opened
+              }
+          }
```

Comment:
```suggestion
          MouseArea {
              anchors.fill: parent
              anchors.margins: -10
              cursorShape: Qt.PointingHandCursor
              onClicked: accountsPopup.open()
          }
```
- is hover needed?
- is additional bool property needed?

Reply:
So I was trying to make additional area to work in same way as changeAccountBtn, which closes popup if pressed again. And if you dont know the state of popup before click happened and dont ignore click event in case it was already opened, it will be close and open instantly.

Reply:
Ok, now I got it, thanks for the clarification :+1:  The solution is a bit hacky and unclear :dizzy_face:  but I don't have a better idea how to solve it.